radio/src/gui/9x/splash.cpp

```diff
@@ -20,18 +20,46 @@
 
 #include "../../opentx.h"
 
+#if !defined GET_FAR_ADDRESS
+#define GET_FAR_ADDRESS(var)                          \
+({                                                    \
+    uint32_t tmp;                                \
+                                                      \
+    __asm__ __volatile__(                             \
+                                                      \
+            "ldi    %A0, lo8(%1)"           "\n\t"    \
+            "ldi    %B0, hi8(%1)"           "\n\t"    \
+            "ldi    %C0, hh8(%1)"           "\n\t"    \
+            "clr    %D0"                    "\n\t"    \
+        :                                             \
+            "=d" (tmp)                                \
+        :                                             \
+            "p"  (&(var))                             \
+    );                                                \
+    tmp;                                              \
+})
+#endif
+
 #if defined(SPLASH)
+
 const pm_uchar splashdata[] PROGMEM = {
   'S','P','S',0,
   #include "bitmaps/9x/splash.lbm"
   'S','P','E',0 };
+  
+#if !defined(PCBMEGA2560)
 const pm_uchar * const splash_lbm = splashdata+4;
+#endif
 
 void drawSplash()
 {
   lcdClear();
-  lcd_img(0, 0, splash_lbm, 0, 0);
-
+#if defined(PCBMEGA2560)
+  lcd_imgfar(0, 0, (pgm_get_far_address(splashdata)+4), 0, 0); //use progmem "far" for splash working with all other options enabled
```

Comment:
Also you will need to define pgm_get_far_address for the simu, perhaps something like 

```c
#define pgm_get_far_address(ptr) ptr
```